### PR TITLE
Update bootstrap to v3

### PR DESCRIPTION
**Pull request by kayneth**
Hi 👋 

I saw the closed PR #1574 and needed to look into API-Platform v3. As I did a test with the bootstrap documentation and needed to update the example code block to make it work for this purpose, is it possible to submit an update?

### CODE REVIEW

File: core/bootstrap.md [hunk 1]
```diff
@@ -139,13 +136,11 @@ use Symfony\Component\Serializer\Normalizer\ProblemNormalizer;
...
 use Symfony\Component\Serializer\Serializer;
 
-/** @deprecated since API Platform 2.7, will be removed in API Platform 3.0 */
-$allowPlainIdentifiers = false;
 $debug = true;
 $defaultContext = [];
 $dataTransformers = [];
 $patchFormats = ['json' => ['application/merge-patch+json'], 'jsonapi' => ['application/vnd.api+json']];
-$formats = ['jsonld' => ['application/ld+json']];
+$formats = ['jsonld' => ['application/ld+json'], 'json' =>  ['application/json']];
```
Comment (alanpoulain) [inline core/bootstrap.md:143]: I don't think this is the default setting.

**Comment by alanpoulain** [comment]
Looks great! @soyuka may have a look at this.

File: core/bootstrap.md (hunk 1 above)
Comment (kayneth) [reply core/bootstrap.md]: You're right, sorry. I was trying something with OpenAPI 😅 

I have taken the values available in `src/Symfony/Bundle/DependencyInjection/Configuration.php` if it's correct for you

**Comment by dunglas** [comment]
Thanks!

**Pull request merged**